src/FSharp.Analyzers.SDK/FSharp.Analyzers.SDK.TestHelpers.fs

```diff
@@ -2,16 +2,149 @@ module FSharp.Analyzers.SDK.TestHelpers
 
 #nowarn "57"
 
+open FSharp.Compiler.Text
+open Microsoft.Build.Logging.StructuredLogger
+open CliWrap
 open System
+open System.IO
 open FSharp.Compiler.CodeAnalysis
-open FSharp.Compiler.Text
 
-let getAnalyzerMsgs source =
-    let loadProject (fcs: FSharpChecker) fileName sourceText =
-        fcs.GetProjectOptionsFromScript(fileName, sourceText)
-        |> Async.RunSynchronously
-        |> fst
+type DotNetVersion =
+    | Six
+    | Seven
```

Comment:
Should probably have an escape hatch for future versions.

Reply:
Especially since the first production-ready release for .Net 8 drops in...a couple hours 😂